frontend/src/components/Details/Personal.vue

```diff
@@ -77,96 +64,148 @@ watch(
             v-model="romUser.now_playing"
             color="romm-accent-1"
             hide-details
-          />
+          >
+            <template v-slot:label
+              ><span>Now playing</span
+              ><span class="ml-2">{{
+                getEmojiForStatus("now_playing")
+              }}</span></template
+            >
+          </v-checkbox>
           <v-checkbox
-            label="Hidden"
             v-model="romUser.hidden"
             color="romm-accent-1"
             hide-details
-          />
+          >
+            <template v-slot:label
+              ><span>Hidden</span
+              ><span class="ml-2">{{
+                getEmojiForStatus("hidden")
+              }}</span></template
+            >
+          </v-checkbox>
         </v-col>
-        <v-col cols="6">
-          <div class="d-flex align-center mt-2">
-            <v-label class="text-body-2 mr-4">Rating</v-label>
-            <v-rating
-              hover
-              ripple
-              length="10"
-              size="32"
-              v-model="romUser.rating"
-              @update:model-value="
-                romUser.rating =
-                  typeof $event === 'number' ? $event : parseInt($event)
-              "
-              active-color="romm-accent-1"
-            />
-          </div>
-          <div class="d-flex align-center mt-3">
-            <v-label class="text-body-2 mr-4">Difficulty</v-label>
-            <v-slider
-              v-model="romUser.difficulty"
-              min="1"
-              max="10"
-              step="1"
-              hide-details
-              track-fill-color="romm-accent-1"
-            />
-            <v-label class="ml-2 opacity-100">
-              {{
-                difficultyEmojis[Math.floor(romUser.difficulty) - 1] ?? "😀"
-              }}</v-label
+        <v-col cols="12" md="7">
+          <v-row
+            class="d-flex align-center"
+            :class="{ 'mt-4': smAndDown }"
+            no-gutters
+          >
+            <v-col cols="12" md="2">
+              <v-label>Rating</v-label>
+            </v-col>
+            <v-col cols="12" md="10">
+              <v-rating
+                :class="{ 'ml-2': mdAndUp }"
+                hover
+                ripple
+                length="10"
+                size="26"
+                v-model="romUser.rating"
+                @update:model-value="
+                  romUser.rating =
+                    typeof $event === 'number' ? $event : parseInt($event)
+                "
+                active-color="romm-accent-1"
+              />
+            </v-col>
+          </v-row>
+          <v-row class="d-flex align-center mt-4" no-gutters>
+            <v-col cols="12" md="2">
+              <v-label>Difficulty</v-label>
+            </v-col>
+            <v-col cols="11" md="9">
+              <v-slider
+                :class="{ 'ml-4': mdAndUp }"
+                v-model="romUser.difficulty"
```

Comment:
Both Difficulty and Completion sliders are triggering API request with each value change, so when the user clicks and drags in the Completion slider, from 0 to 100, we are sending 100 API requests to the backend.

This could be fixed in a separate PR, but we could find a way to trigger a single update.

Reply:
Hmm yeah you are right, we should be able to just detect the click release or the touch release. I'll invesyit for the next PR